### PR TITLE
Feature 16/seb

### PR DESCRIPTION
**Pull request by Seb105**
Relates to https://github.com/CntoDev/compositions/issues/140.

Adds a dependency to zen_dialog.

Obviously only to be implemented when zeus_enhacned gets into the repo.

### CODE REVIEW

File: addons/functions/config.cpp [hunk 1]
```diff
@@ -5,7 +5,9 @@ class CfgPatches {
     class cnto_functions {
         units[] = {};
         weapons[] = {};
-        requiredAddons[] = {};
+        requiredAddons[] = {
+			"zen_dialog"
+		};
```
Comment (freghar) [inline addons/functions/config.cpp]: Please fix the indent at least in the shared `config.cpp`, spaces, not tabs.

**Review by milivojm — COMMENTED** [review]
Do you guys get "purchase DLC" message in inventory?

**Comment by milivojm** [comment]
This works. Please fix formatting and we can merge it when Zen is introduced in modset.

**Review by milivojm — APPROVED** [review]
Approved, on hold till Zen.

**Comment by freghar** [comment]
I think you can merge it even now - these new cnto-additions won't be included (AFAIK) without the new version of my arma-additions, which already require ZEN, so we'll already have ZEN by the time this codebase is used.

**Pull request closed without merging**